src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -1763,9 +1763,24 @@ private bool OnTouchEventForTrack(object source, TouchEventArgs e)
                     valueIndicatorImage.Show();
                 }
 
+                if (null != sliderSlidingStartedHandler)
+                {
+                    SliderSlidingStartedEventArgs args = new SliderSlidingStartedEventArgs();
+                    args.CurrentValue = curValue;
+                    sliderSlidingStartedHandler(this, args);
+                }
+
                 Vector2 pos = e.Touch.GetLocalPosition(0);
                 CalculateCurrentValueByTouch(pos);
                 UpdateValue();
+            }
+            else if (state == PointStateType.Up ||
+                    state == PointStateType.Leave) // When the thumb consumes touch event, the track gets leave point type.
+            {
```

Comment:
It might be very strange usecase but anyway, we might think about Down -> Leave -> (goto inside again) --> Leave again --> (goto inside again) --> blabla --> Finished
(Unfortunatly, we don't have "ReEntry" state now.... maybe?)


That mean, there can comes 1 Down + unlimitied Leave events come.

I think we should consider about isPressed` here.

Continous of this thing, I think we'd better use `UpdateState(isFocused, true / false)` function near here
```suggestion
            {
                if(isPressed) continue;
```

Reply:
oh very good point!

Reply:
Added `UpdateState()` to update the pressed state on Touch event.